lib/StaticNarrative/util.py

```diff
@@ -0,0 +1,38 @@
+"""This module contains some utility functions for the SetAPI."""
+
+import os
+import re
+
+from installed_clients.DataFileUtilClient import DataFileUtil
+
+
+def check_reference(ref) -> bool:
```

Comment:
This is rewritten in so many places throughout KBase that Python should just make it a built-in function.